docs/source/teams/roles_and_permissions.rst

```diff
@@ -105,14 +139,28 @@ Admins and users with the **Can manage** permission on a dataset can configure
 a dataset's permissions under the dataset's
 :ref:`Manage tab <teams-managing-datasets>` in the FiftyOne Teams App.
 
-A user's permissions on a dataset is the maximum of their permissions from the
+In FiftyOne Teams, dataset permissions for a user are determined by both the 
+access they receive from their group's permissions and individual permissions 
+assigned to them. This tiered permissions system is 
```

Comment:
nit: don't think the `This tiered ...` sentence is necessary here as you already explained this at the beginning of the `Groups` section